### PR TITLE
Adapt Help Command and its relevant FXML display

### LINKED ISSUE #152: Add  Help to UML diagram, Java class and test file

### PR DESCRIPTION
**Pull request by Cherweijie**
#152

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,8 +15,55 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://bit.ly/2U9MBrf";
+    public static final String HELP_MESSAGE = "Delino supports add, clear, delete, done, edit, exit, find, help,"
+            + " import, list, return, sort and undo commands.\n"
```
Comment (JeremyLoh) [inline src/main/java/seedu/address/ui/HelpWindow.java]: I think there is no more sort command. It has been changed to nearby. Also, the add command has been changed to insert. The updated commands should be in the Command summary in the [User Guide](https://ay1920s2-cs2103t-w12-1.github.io/main/UserGuide.html)
Kenny has also requested to change the find command to search command as shown here #177

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -15,8 +15,55 @@
...
+
+            + "\n 10. Display the default list of orders or a list based on specified status."
+            + "\n      Eg. list OR list done OR list undone \n"
+
+            + "\n 11. Create a return order and adds it into the list of returns."
+            + "\n      Eg. return `tid/ac17s2a n/Bobby Tan a/123 Delta Road #03-333, Singapore 123456"
+            + " p/91230456 ts/12-12-2020 w/Jurong Warehouse \n"
+
+            + "\n 12. Sort the list of orders by the specified sequence."
+            + "\n      Eg. sort ts/ (Sorts the orders in ascending order based on timestamp of the orders \n"
```
Comment (JeremyLoh) [inline src/main/java/seedu/address/ui/HelpWindow.java]: The sort command has been changed to nearby.

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 3]
```diff
@@ -15,8 +15,55 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://bit.ly/2U9MBrf";
```
Comment (JeremyLoh) [inline src/main/java/seedu/address/ui/HelpWindow.java]: The link here https://bit.ly/2U9MBrf
refers to https://deploy-preview-86--ay1920s2-cs2103t-w12-1.netlify.com/userguide.
I think it should be linked to 
https://ay1920s2-cs2103t-w12-1.github.io/main/UserGuide.html
It's where our user guide is at (for the master branch).

**Review by JeremyLoh — CHANGES_REQUESTED** [review]
This PR should be good to merge after changing the Help UserGuide URL and command info based on the user guide here: https://ay1920s2-cs2103t-w12-1.github.io/main/UserGuide.html
Thanks!

**Review by khsc96 — CHANGES_REQUESTED** [review]
Do changes mentioned by Jeremy thanks!

**Comment by Cherweijie** [comment]
@JeremyLoh @khsc96 changes have been done, please review them. Thank you!

**Review by khsc96 — APPROVED** [review]
Well done! LGTM

**Review by JeremyLoh — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**